loopy/target/c/__init__.py

```diff
@@ -648,6 +648,37 @@ def with_types(self, arg_id_to_dtype, callables_table):
                             0: NumpyType(dtype),
                             -1: NumpyType(np.int32)}),
                     callables_table)
+        elif name in ["bessel_jn", "bessel_yn"]:
```

Comment:
I think I'd prefer if this were a GNU-specific subclass.

Reply:
Thanks, done!